templates/pages/2fa/2fa_status.html.twig

```diff
@@ -0,0 +1,133 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2022 Teclib' and contributors.
+ # @copyright 2003-2014 by the INDEPNET Development Team.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+<form method="post" action="#" data-submit-once>
+   <div class="alert alert-info">
+      <h1>{{ __('2FA Enabled') }}</h1>
+      <h2>
+         {% if enforcement == constant('Glpi\\Security\\TOTPManager::ENFORCEMENT_MANDATORY_GRACE_PERIOD') %}
+            {{ __('2FA grace period ends %s')|format(grace_period_end|relative_datetime) }}
+         {% elseif enforcement == constant('Glpi\\Security\\TOTPManager::ENFORCEMENT_MANDATORY') %}
+            {{ __('Mandatory') }}
+         {% else %}
+            {{ __('Optional') }}
+         {% endif %}
+      </h2>
+   </div>
+   <div id="backup_codes" class="d-none py-2" style="width: fit-content;">
+      <h3 class="mb-2">{{ __('Backup codes (This is the only time these will be shown)') }}</h3>
```

Comment:
Maybe the `This is the only time these will be shown` message should be shown in a :warning: block.